### PR TITLE
More clock options

### PR DESCRIPTION
**Pull request by unsoluble**
Should have links to both bgclock.html and clock-color.html in the main link panel. Might be a tidier way of putting them both on one line, but didn't want to mess with the CSS.

### CODE REVIEW

**Comment by unsoluble** [comment]
When I submitted this I was thinking only `bgclock` and `clock-color` need to be in here, but then when I was adding them to the Readme figured all three (including `clock`) ought to be in *there*, so maybe all three should be in here too. Just a little wary of it getting cluttered.

Tweak the styling perhaps so we can put all three view options on one line?

**Comment by unsoluble** [comment]
Updated the nav to show all three clocks in one row. Looks like this:
screen shot 2018-02-11 at 6 33 09 pm

Conflict with the dev progress on language.js there should be easily resolved; stems from the change to the plural "Clock Views".

**Comment by unsoluble** [comment]
Any chance someone could offer a hand in finalizing this one? I don't think I'm able to resolve the language.js conflicts, or at least I'm not sure how to do that on my end. Should be an easy fix, and then this one could be merged.

**Comment by PieterGit** [comment]
@unsoluble I resolved the merge errors for you. Is this ready to merge? LGTM but I didn't test it yet.

**Comment by unsoluble** [comment]
Thanks! I think it should be good to go, yep.

**Comment by PieterGit** [comment]
Ok, I'll test it myself this week and have a look at why this PR adds language lines. Then I merge it, unless @sulkaharo or @MilosKozak beat me.

**Comment by PieterGit** [comment]
@unsoluble according to @sulkaharo there are some issues with this PR with mobile devices, see
See https://gitter.im/nightscout/public?at=5b42401dbd92d807829dfce8
This PR might be postponed to 0.11 release, because we want to release 0.10.3 soon.

**Comment by unsoluble** [comment]
Could try to fix if I had more information — not clear what the concern is.

**Comment by unsoluble** [comment]
This is what it looks like on an iPhone SE, which has one of the smallest screens on the market right now: 
img_0010

File: views/index.html [hunk 1]
```diff
@@ -169,7 +169,12 @@
                   <li class="profilecontrol"><a id="editprofilelink" href="profile" target="profileeditor" class="translate">Profile Editor</a></li>
                   <li class="foodcontrol"><a id="editfoodlink" href="food" target="foodeditor" class="translate">Food Editor</a></li>
                   <li><a id="admintoolslink" href="admin" target="admintools" class="translate">Admin Tools</a></li>
-                  <li><a id="clocklink" href="bgclock.html" target="admintools" class="translate">Clock View</a></li>
+                  <li class="multilink">
+                  	<a>Clock Views:</a>
```
Comment (PieterGit) [inline views/index.html]: The Clock Views part is not translatable

**Comment by PieterGit** [comment]
Small issue: `Clock Views: ` is not translatable. It shows fine with Samsung S7 and with Windows 10 and Chrome. I had bad luck with testing this, see other problems at https://github.com/nightscout/cgm-remote-monitor/pull/3709

**Comment by unsoluble** [comment]
I don’t know what you mean by translatable. Can you elaborate?

**Comment by PieterGit** [comment]
@unsoluble I added translation strings for it, but it does not translate. See
https://github.com/nightscout/cgm-remote-monitor/pull/3709/files#diff-29e70f5c52b22198247a7936ea3f1d7dR11476

image

I expected: Klokweergave instead of Clock Views

**Comment by unsoluble** [comment]
Oh I see it: should be a class=translate in that <a> tag. Can you add that? Would myself but will be on my phone only for a while.

**Comment by unsoluble** [comment]
This conflict-resolving stuff is new to me; I'm not sure if I'm approaching it the right way. Tried to just commit with the updated contents of language.js that you linked, but that doesn't seem to have done the trick. Is there something else I need to do on my end?

**Comment by PieterGit** [comment]
Merged with https://github.com/nightscout/cgm-remote-monitor/pull/3712
Closing PR

**Pull request merged**